.travis.yml

```diff
@@ -25,6 +25,7 @@ install:
   - rvm use 2.2.3 --install --fuzzy
   - gem update --system
   - gem install sass
+  - gem install ruby_dep -v 1.3.1
```

Comment:
Already fixed in https://github.com/julien-truffaut/newts/pull/24/commits/49a172f5fa94e869ca79f93eed10b68fceb9e51d, no need to install an old ruby_dep.